python_benchmarks/20_polars_lazy.py

```diff
@@ -1,18 +1,15 @@
-import polars
+import polars as pl
 
 
 def polars_lazy(fname):
-    dtypes = {f"column{i}": (polars.UInt32 if i == 0 else polars.Float32())
+    dtypes = {f"column{i}": (pl.UInt32 if i == 0 else pl.Float32())
               for i in range(9)}
-    return (polars.scan_csv(fname, dtypes=dtypes)
-                  .select(
-                      polars.sum_horizontal(polars.all()
-                                                  .exclude("0")
+    return (pl.scan_csv(fname, dtypes=dtypes)
+                  .filter(pl.sum_horizontal(pl.all()
```

Comment:
Doing this in the `filter` allows Polars to push this predicate completely down to the scan level. And is more similar to what is done in the SQL query. E.g. the full predicate is in the `where` clause. So it is fair to give polars that information as well.